webapi/webapi-nfc-w3c-tests/nfc/support/nfc_common.js

```diff
@@ -29,7 +29,12 @@ Authors:
 
 */
 
-document.write("<script language=\"javascript\" src=\"..\/resources\/testharness.js\"><\/script>");
-document.write("<script language=\"javascript\" src=\"..\/resources\/testharnessreport.js\"><\/script>");
+var head_src = document.head.innerHTML;
+if (head_src.search(/\/testharness.js\W/) === -1) {
```

Comment:
Hmm, if the `nfc_common.js` script is loaded in the head and before `testharness` scripts loaded, this may not work. See http://stackoverflow.com/questions/9849204/javascript-in-html-head-innerhtml-not-working

Example:
```
<head>
  <script src="support/nfc_common.js"></script>
  <script src="../resources/testharness.js"></script>
</head>
```

Another, why not use `String.indexOf()` rather than `String.search()`?